### DISCUSSION TITLE
Change URL in E-Mail Notification

### DISCUSSION THREAD

**Discussion by BEBU88** [immich-app/immich#10456] [Q&A] [answered]
Hi,

I just configured the new Mail-Notifications. Is there a way to change the hostname and port? Actually the text ist 'Hey Admin, this is the test email from your Immich Instance
http://localhost:2283/'. Obviously my Instance is not reachable via localhost. Is there a way to customize my domain and change it to https:// with no port cause I'm using a reverse proxy in front of Immich?

**Comment by bo0tzz** [comment, marked as answer]
Admin > Settings > Server settings > External domain

**Reply by BEBU88** [reply]
Oh, totally missed that... Thank You.
